wire-ios/Wire-iOS Tests/UserCellTests.swift

```diff
@@ -16,18 +16,20 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-@testable import Wire
 import WireUtilities
 import XCTest
 
+@testable import Wire
+
 final class UserCellTests: BaseSnapshotTestCase {
 
     // MARK: - Properties
 
-    var sut: UserCell!
-    var teamID = UUID()
-    var conversation: MockGroupDetailsConversation!
-    var mockUser: MockUserType!
+    private var sut: UserCell!
+    private var teamID = UUID()
+    private var conversation: MockGroupDetailsConversation!
+    private var mockUser: MockUserType!
+    private let helper = SnapshotHelper()
```

Comment:
optional: Maybe we can rename all occurrences of this identifier. (in all files)
```suggestion
    private let snapshotHelper = SnapshotHelper()
```

Reply:
Done!